### ISSUE TITLE
In the latest version it does not show metadata

### ISSUE THREAD

**Issue by webma5ter23** [AzuraCast/AzuraCast#5260]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

0.15.2

### What happened?

In the panel and the streaming signal it does not show the metadata (names of songs) that it sends from the encoder software (Butt) but I have also tried with Altacast and it does not work neither in AAC OR MP3.

This started to happen since Azuracast was updated, before it was showing fine.

Any solution?

![Captura de pantalla 2022-04-05 210748](https://user-images.githubusercontent.com/41419364/161871541-8003b608-1db5-401d-8d4b-428c3d399c14.png)
![Captura de pantalla 2022-04-05 210829](https://user-images.githubusercontent.com/41419364/161871542-42382dbd-8dc5-4605-a150-26902edb109a.png)

### Relevant log output

```shell
2022/04/05 20:42:29 [lang:3] DJ Source connected! Last authenticated DJ: La Vereda WebRadio - [("content-type", "application/ogg"), ("icy-name", "La Vereda WebRadio"), ("icy-genre", "Rock"), ("icy-url", "https://laveredawebradio.com"), ("icy-pub", "1"), ("icy-irc", "N/A"), ("icy-icq", "N/A"), ("icy-aim", "N/A"), ("icy-br", "128")]
2022/04/05 20:42:29 [lang:3] API djon - Sending POST request to 'http://web/api/internal/2/djon' with body: { "user": "La Vereda WebRadio" }
2022/04/05 20:42:29 [lang:3] API djon - Response (200): true
2022/04/05 20:42:29 [input_streamer:3] Decoding...
2022/04/05 20:42:29 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "cristian", "user": "La Vereda WebRadio" }
2022/04/05 20:42:30 [lang:3] API auth - Response (200): true
[mp3float @ 0x7f9910c8b200] Header missing
2022/04/05 20:43:32 [decoder:2] Decoding "/var/azuracast/stations/la_vereda_webradio/media/Musica/lenny_kravitz_-_fly_away.mp3" ended: Ffmpeg_decoder.End_of_file.
[mp3 @ 0x7f99113bd440] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f99113bd440] Estimating duration from bitrate, this may be inaccurate
2022/04/05 20:43:32 [next_song:3] Prepared "/var/azuracast/stations/la_vereda_webradio/media/Musica/michael_jackson-2C_justin_timberlake_love_never_felt_so_good.mp3" (RID 1).
2022/04/05 20:43:32 [cue_next_song:3] Cueing in...
2022/04/05 20:43:32 [lang:3] API nextsong - Sending POST request to 'http://web/api/internal/2/nextsong' with body: 
2022/04/05 20:43:32 [crossfade_0:3] Analysis: -infdB / -12.515219dB (2.96s / 2.96s)
2022/04/05 20:43:32 [crossfade_0:3] No transition: using default.
2022/04/05 20:43:32 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:43:32 [lang:3]   "song_id": "84413d810d2029b2fe5e2804c0d9e28b",
2022/04/05 20:43:32 [lang:3]   "playlist_id": "4",
2022/04/05 20:43:32 [lang:3]   "media_id": "76"
2022/04/05 20:43:32 [lang:3] }
2022/04/05 20:43:32 [lang:3] API nextsong - Response (200): annotate:title="enanitos verdes",artist="luz de dia",duration="272.93",song_id="2667123672e5d707c5335694c31ca033",media_id="190",liq_amplify="0.00dB",playlist_id="4":/var/azuracast/stations/la_vereda_webradio/media/Musica/luz_de_dia_-_enanitos_verdes.mp3
2022/04/05 20:43:33 [lang:3] API feedback - Response (200): OK
2022/04/05 20:43:35 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:43:35 [lang:3]   "song_id": "3f454003d79706c85028af99a86aa9cf",
2022/04/05 20:43:35 [lang:3]   "playlist_id": "4",
2022/04/05 20:43:35 [lang:3]   "media_id": "200"
2022/04/05 20:43:35 [lang:3] }
2022/04/05 20:43:35 [lang:3] API feedback - Response (200): OK
2022/04/05 20:43:41 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "La%20Vereda%20WebRadio%3Acristian", "user": "source" }
2022/04/05 20:43:41 [lang:3] API auth - Response (403): false
2022/04/05 20:43:49 [live_fallback:3] Switch to map_metadata_0 with transition.
2022/04/05 20:44:36 [live_fallback:3] Switch to crossfade_0 with forgetful transition.
2022/04/05 20:44:36 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:44:36 [lang:3]   "song_id": "3f454003d79706c85028af99a86aa9cf",
2022/04/05 20:44:36 [lang:3]   "playlist_id": "4",
2022/04/05 20:44:36 [lang:3]   "media_id": "200"
2022/04/05 20:44:36 [lang:3] }
2022/04/05 20:44:36 [lang:3] API feedback - Response (200): OK
2022/04/05 20:45:59 [live_fallback:3] Switch to map_metadata_0 with transition.
2022/04/05 20:46:46 [live_fallback:3] Switch to crossfade_0 with forgetful transition.
2022/04/05 20:46:46 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:46:46 [lang:3]   "song_id": "3f454003d79706c85028af99a86aa9cf",
2022/04/05 20:46:46 [lang:3]   "playlist_id": "4",
2022/04/05 20:46:46 [lang:3]   "media_id": "200"
2022/04/05 20:46:46 [lang:3] }
2022/04/05 20:46:47 [lang:3] API feedback - Response (200): OK
2022/04/05 20:48:10 [live_fallback:3] Switch to map_metadata_0 with transition.
2022/04/05 20:48:56 [live_fallback:3] Switch to crossfade_0 with forgetful transition.
2022/04/05 20:48:56 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:48:56 [lang:3]   "song_id": "3f454003d79706c85028af99a86aa9cf",
2022/04/05 20:48:56 [lang:3]   "playlist_id": "4",
2022/04/05 20:48:56 [lang:3]   "media_id": "200"
2022/04/05 20:48:56 [lang:3] }
2022/04/05 20:48:56 [lang:3] API feedback - Response (200): OK
2022/04/05 20:49:56 [decoder:2] Decoding "/var/azuracast/stations/la_vereda_webradio/media/Musica/michael_jackson-2C_justin_timberlake_love_never_felt_so_good.mp3" ended: Ffmpeg_decoder.End_of_file.
2022/04/05 20:49:57 [next_song:3] Prepared "/var/azuracast/stations/la_vereda_webradio/media/Musica/luz_de_dia_-_enanitos_verdes.mp3" (RID 3).
2022/04/05 20:49:57 [cue_next_song:3] Cueing in...
2022/04/05 20:49:57 [lang:3] API nextsong - Sending POST request to 'http://web/api/internal/2/nextsong' with body: 
2022/04/05 20:49:57 [crossfade_0:3] Analysis: -25.149098dB / -19.533571dB (2.96s / 2.96s)
2022/04/05 20:49:57 [crossfade_0:3] new >= old + margin, old >= medium and new <= high.
2022/04/05 20:49:57 [crossfade_0:3] New source is significantly louder than old one.
2022/04/05 20:49:57 [crossfade_0:3] Transition: crossed, fade-out.
2022/04/05 20:49:57 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:49:57 [lang:3]   "song_id": "2667123672e5d707c5335694c31ca033",
2022/04/05 20:49:57 [lang:3]   "playlist_id": "4",
2022/04/05 20:49:57 [lang:3]   "media_id": "190"
2022/04/05 20:49:57 [lang:3] }
2022/04/05 20:49:57 [lang:3] API nextsong - Response (200): annotate:title="Knocking haven door",artist="f130G´n´R",duration="336.62",song_id="ec748b63672460f73d77baba55b989a9",media_id="69",liq_amplify="0.00dB",playlist_id="4":/var/azuracast/stations/la_vereda_webradio/media/Musica/guns_n_roses_-_knocking_haven_door.mp3
2022/04/05 20:49:57 [lang:3] API feedback - Response (200): OK
[mp3 @ 0x7f990c00b140] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f990c00b140] Estimating duration from bitrate, this may be inaccurate
2022/04/05 20:49:59 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:49:59 [lang:3] API auth - Response (403): false
2022/04/05 20:50:00 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:00 [lang:3] API auth - Response (403): false
2022/04/05 20:50:01 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:01 [lang:3] API auth - Response (403): false
2022/04/05 20:50:02 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:02 [lang:3] API auth - Response (403): false
2022/04/05 20:50:03 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:03 [lang:3] API auth - Response (403): false
2022/04/05 20:50:04 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:04 [lang:3] API auth - Response (403): false
2022/04/05 20:50:05 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:05 [lang:3] API auth - Response (403): false
2022/04/05 20:50:06 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:06 [lang:3] API auth - Response (403): false
2022/04/05 20:50:06 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:07 [lang:3] API auth - Response (403): false
2022/04/05 20:50:11 [input_streamer:2] Error while reading from client: Connection reset by peer in read()
2022/04/05 20:50:11 [lang:3] API djoff - Sending POST request to 'http://web/api/internal/2/djoff' with body: { "user": "La Vereda WebRadio" }
Invalid return value 0 for stream protocol
2022/04/05 20:50:11 [lang:3] API djoff - Response (200): true
Invalid return value 0 for stream protocol
2022/04/05 20:50:11 [input_streamer:2] Feeding stopped: Ffmpeg_decoder.End_of_file.
2022/04/05 20:50:11 [lang:3] API djoff - Sending POST request to 'http://web/api/internal/2/djoff' with body: { "user": "" }
2022/04/05 20:50:11 [lang:3] API djoff - Response (200): true
2022/04/05 20:50:15 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:16 [lang:3] API auth - Response (403): false
2022/04/05 20:50:16 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:17 [lang:3] API auth - Response (403): false
2022/04/05 20:50:17 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:17 [lang:3] API auth - Response (403): false
2022/04/05 20:50:18 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:18 [lang:3] API auth - Response (403): false
2022/04/05 20:50:19 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:19 [lang:3] API auth - Response (403): false
2022/04/05 20:50:19 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:20 [lang:3] API auth - Response (403): false
2022/04/05 20:50:20 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:21 [lang:3] API auth - Response (403): false
2022/04/05 20:50:21 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:22 [lang:3] API auth - Response (403): false
2022/04/05 20:50:22 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:22 [lang:3] API auth - Response (403): false
2022/04/05 20:50:23 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:23 [lang:3] API auth - Response (403): false
2022/04/05 20:50:24 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:24 [lang:3] API auth - Response (403): false
2022/04/05 20:50:25 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:25 [lang:3] API auth - Response (403): false
2022/04/05 20:50:25 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:26 [lang:3] API auth - Response (403): false
2022/04/05 20:50:26 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:27 [lang:3] API auth - Response (403): false
2022/04/05 20:50:27 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:28 [lang:3] API auth - Response (403): false
2022/04/05 20:50:28 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:28 [lang:3] API auth - Response (403): false
2022/04/05 20:50:29 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:29 [lang:3] API auth - Response (403): false
2022/04/05 20:50:30 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:30 [lang:3] API auth - Response (403): false
2022/04/05 20:50:31 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:31 [lang:3] API auth - Response (403): false
2022/04/05 20:50:31 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:32 [lang:3] API auth - Response (403): false
2022/04/05 20:50:32 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:33 [lang:3] API auth - Response (403): false
2022/04/05 20:50:33 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:34 [lang:3] API auth - Response (403): false
2022/04/05 20:50:34 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:34 [lang:3] API auth - Response (403): false
2022/04/05 20:50:35 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:35 [lang:3] API auth - Response (403): false
2022/04/05 20:50:36 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:36 [lang:3] API auth - Response (403): false
2022/04/05 20:50:37 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:37 [lang:3] API auth - Response (403): false
2022/04/05 20:50:37 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:38 [lang:3] API auth - Response (403): false
2022/04/05 20:50:38 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:39 [lang:3] API auth - Response (403): false
2022/04/05 20:50:39 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:39 [lang:3] API auth - Response (403): false
2022/04/05 20:50:40 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:40 [lang:3] API auth - Response (403): false
2022/04/05 20:50:41 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:41 [lang:3] API auth - Response (403): false
2022/04/05 20:50:42 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:42 [lang:3] API auth - Response (403): false
2022/04/05 20:50:42 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:43 [lang:3] API auth - Response (403): false
2022/04/05 20:50:43 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:44 [lang:3] API auth - Response (403): false
2022/04/05 20:50:44 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:50:44 [lang:3] API auth - Response (200): true
2022/04/05 20:50:45 [lang:3] DJ Source connected! Last authenticated DJ: La Vereda WebRadio - [("Host", "srv2.radiostreamuy.com:8015"), ("icy-name", "La Vereda WebRadio"), ("icy-url", "https://laveredawebradio.com"), ("icy-pub", "0"), ("icy-br", "128"), ("content-type", "audio/mpeg")]
2022/04/05 20:50:45 [lang:3] API djon - Sending POST request to 'http://web/api/internal/2/djon' with body: { "user": "La Vereda WebRadio" }
2022/04/05 20:50:45 [lang:3] API djon - Response (200): true
2022/04/05 20:50:45 [input_streamer:3] Decoding...
2022/04/05 20:50:49 [live_fallback:3] Switch to map_metadata_0 with transition.
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
2022/04/05 20:53:28 [input_streamer:2] Feeding stopped: Ffmpeg_decoder.End_of_file.
2022/04/05 20:53:28 [lang:3] API djoff - Sending POST request to 'http://web/api/internal/2/djoff' with body: { "user": "La Vereda WebRadio" }
2022/04/05 20:53:28 [lang:3] API djoff - Response (200): true
2022/04/05 20:53:55 [lang:3] API auth - Sending POST request to 'http://web/api/internal/2/auth' with body: { "password": "djcristian", "user": "La Vereda WebRadio" }
2022/04/05 20:53:55 [lang:3] API auth - Response (200): true
2022/04/05 20:53:55 [lang:3] DJ Source connected! Last authenticated DJ: La Vereda WebRadio - [("Host", "srv2.radiostreamuy.com:8015"), ("icy-name", "La Vereda WebRadio"), ("icy-url", "https://laveredawebradio.com"), ("icy-pub", "0"), ("icy-br", "64"), ("content-type", "audio/aac")]
2022/04/05 20:53:55 [lang:3] API djon - Sending POST request to 'http://web/api/internal/2/djon' with body: { "user": "La Vereda WebRadio" }
2022/04/05 20:53:55 [lang:3] API djon - Response (200): true
2022/04/05 20:53:55 [input_streamer:3] Decoding...
2022/04/05 20:53:58 [live_fallback:3] Switch to crossfade_0 with transition.
2022/04/05 20:53:58 [lang:3] API feedback - Sending POST request to 'http://web/api/internal/2/feedback' with body: {
2022/04/05 20:53:58 [lang:3]   "song_id": "2667123672e5d707c5335694c31ca033",
2022/04/05 20:53:58 [lang:3]   "playlist_id": "4",
2022/04/05 20:53:58 [lang:3]   "media_id": "190"
2022/04/05 20:53:58 [lang:3] }
2022/04/05 20:53:58 [lang:3] API feedback - Response (200): OK
2022/04/05 20:54:25 [live_fallback:3] Switch to map_metadata_0 with transition.
```

**Comment by SlvrEagle23** [comment]
@webma5ter23 This is a known issue with the first metadata update received after a live broadcast; we've simply updated the code to say "Live Broadcast" instead of the previous song's metadata to ensure users know that the song has changed.

The easiest fix for this is to have the DJ immediately send a metadata update after connecting. A lot of stations do this by playing a quick station identifying clip, and then another one a few seconds later, to trigger the necessary metadata update.

Unfortunately, because this issue is caused by Liquidsoap, an upstream piece of software that we don't directly maintain, there isn't anything else we can do to resolve it. We've worked with them on the issue before, but have been unable to resolve it for all use cases.
